cmd/admin-heal-ops.go

```diff
@@ -689,8 +728,7 @@ func (h *healSequence) healBucket(bucket string) error {
 		return nil
 	}
 
-	if err = objectAPI.HealObjects(h.ctx, bucket,
-		h.objPrefix, h.healObject); err != nil {
+	if err := objectAPI.HealObjects(h.ctx, bucket, h.objPrefix, h.healObject); err != nil {
```

Comment:
Is HealObjects() still useful? since it's still using older listing style.

Reply:
> Is HealObjects() still useful? since it's still using older listing style.

This is still used for the manual healing, when mc admin heal passes a specific bucket + prefix to heal those objects